docs/components/Text-utilities/text-util.css

```diff
@@ -0,0 +1,42 @@
+
+.heading-one {
+    font-size: 2.7rem;
+}
+
+.heading-two {
+    font-size: 2.4rem;
+}
+
+.heading-three {
+    font-size: 2.1rem;
+}
+
+.heading-four {
+    font-size: 1.8rem;
+}
+
+.heading-five {
+    font-size: 1.5rem;
+}
+
+.heading-six {
+    font-size: 1.2rem;
+}
+
+.small-text {
```

Comment:
you can use rem for fontsize too.

Reply:
It was 0.9 rem so i switched to px to make it clean